lib/modules/manager/gradle/extract.ts

```diff
@@ -53,6 +60,9 @@ export async function extractAllPackageFiles(
       deps: [],
     };
 
+    // Check if gradle-consistent-versions plugin is in use by repo
+    const usesConsistentVersionPlugin = usesGcv(reorderedFiles, fileContents);
```

Comment:
Ah, my bad, it was supposed to be ABOVE the for-look to do the check exactly once. If I move it inside `isGcvPropsFile` then it will be ran twice, once for the `.props` file and once for the `.lock` file.

So I propose to move it up anove the for-loop, line 56 instead?

Reply:
Please move it under `} else if (isGcvPropsFile(packageFile)) {`

Having it above the for loop will unconditionally run a bunch of operations that check for ` 'com.palantir.consistent-versions'`. This is surely relevant for Gcv but not for the vast majority of Gradle projects which don't use this plugin. Minimal impact on these projects can only be warranted if the checks run just in case `versions.props` exists.

Reply:
As stated elsewhere, this is all reworked now..